src/components/carousel/embla-card-carousel.tsx

```
@@ -107,33 +184,115 @@ export function EmblaCardCarousel({
+                ? node.key
+                : `${fallbackKeyPrefix}-${idx}`;
+
+            return (
+              <div
+                key={nodeKey}
+                className={cn(
+                  'box-border min-w-0 cursor-pointer pl-(--carousel-gap)',
+                  'flex-[0_0_calc(100%/var(--carousel-per-view))]',
+                )}
+                aria-roledescription="slide"
+              >
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-roledescription' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)